src/app/component/auth/components/sign-up/sign-up.validator.ts

```diff
@@ -25,8 +25,12 @@ export function ValidatorRegExp(controlName: string) {
     if (control.value === '') {
       control.setErrors({ required: true });
     } else {
-      if (!control.value.match(regexp)) {
+      if (controlName === 'password' && control.value.length < 8 ) {
+        control.setErrors({ minlength: true });
+
+      } else if (!control.value.match(regexp)) {
         control.setErrors({ symbolInvalid: true });
+
```

Comment:
I removed extra space in that line  if (controlName === 'password' && control.value.length < 8 ) {
But there  - control.setErrors({ symbolInvalid: true }); I don't change, because it's general style in the project.
One space after and before the curly braces.

Reply:
i mean line 33 and 30

Reply:
Remove line 30 and 33.